docs/hub-deployment-guide/deploy-support/setup-grafana-dashboards.md

```diff
@@ -0,0 +1,68 @@
+(setup-grafana)=
+# Setup grafana dashboards
+
+Now, we will deploy some specific dashboards to the grafana instance we deployed as part of [](deploy-support-chart).
+
+````{seealso}
+It is also possible to [enable logging in with GitHub](grafana-dashboards:github-auth)
+to allow Community Representatives and Hub Administrators to access these dashboards as well.
+```{warning}
+We should enable GitHub login for _all_ Grafana dashboards running on **dedicated**
+clusters, so the Community Representatives have access to them.
+```
+````
```

Comment:
I'm going to go out on a limb here and say that this is a good policy to have regarding grafana access 😄

Reply:
Yes I agree. Long term it might be nice to instead have it use the JupyterHub for auth, but this is easier to do for now :)

Reply:
Having just setup this up for the first time, I see a need for pushing forward the Hub OAuth flow. _Unless_ we are fine with either i) _anyone_ being able to "sign up" and access the grafana dashbaords, or ii) also creating a grafana-viewers GitHub team as part of https://github.com/2i2c-org/infrastructure/issues/1806

ref: https://github.com/2i2c-org/infrastructure/pull/1830#discussion_r1007924131